rcCollectionroutes.js

```diff
@@ -0,0 +1,67 @@
+var express = require('express');
+
+var bodyParser = require('body-parser');
+
+var urlEncoded = bodyParser.urlencoded({ extended: false });
+
+var rcCollection = require('../models/rcCollectionmodel');
```

Comment:
updated

From: jamdon 
Sent: Friday, June 17, 2016 1:40 AM
To: JSCRIPT300-Spring2016/Final_Project 
Cc: seancava ; Author 
Subject: Re: [JSCRIPT300-Spring2016/Final_Project] Final Project Proposal (#10)

In rcCollectionroutes.js:

> @@ -0,0 +1,67 @@
> +var express = require('express');
> +
> +var bodyParser = require('body-parser');
> +
> +var urlEncoded = bodyParser.urlencoded({ extended: false });
> +
> +var rcCollection = require('../models/rcCollectionmodel');
The path you're using to require in this module is incorrect.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.